src/types.ts

```diff
@@ -1,6 +1,9 @@
-export type Validator<T = any, E = any> = (subject: T | undefined, context?: object) => E | undefined
-export type SubjectType<V extends Validator> = Parameters<V>[0]
-
-export type PrimitiveErrors = Array<string>
-export type ObjectTopLevelError<E> = { _self: E }
+// deno-lint-ignore ban-types no-explicit-any
+export type Validator<T = any, E = any, C extends {} = {}> = (
+  subject: T | undefined,
+  context?: C,
+) => E | undefined;
+export type SubjectType<V extends Validator> = Parameters<V>[0];
```

Comment:
`V` => `Validator`, This is probably the example case, why most people write type parameters in Uppercase though (`VALIDATOR`)

Reply:
Will change.